server/src/main/java/org/elasticsearch/action/support/nodes/TransportNodesAction.java

```diff
@@ -138,6 +138,12 @@ protected void resolveRequest(NodesRequest request, ClusterState clusterState) {
         request.setConcreteNodes(Arrays.stream(nodesIds).map(clusterState.nodes()::get).toArray(DiscoveryNode[]::new));
     }
 
+    /**
+     * Get a backwards compatible transport action name
+     */
+    protected String getBwcTransportNodeAction(DiscoveryNode node) {
```

Comment:
I think this just should be 
```suggestion
    protected String getTransportNodeAction(DiscoveryNode node) {
```
I don't think the "BWC" helps with the method name - it assumes that this is only relevant for BWC, but really it's "what is the right action for this node" ?

Reply:
Also, I think this part of the change should go to master - I know you don't need it for BWC purposes there, but we shouldn't make core API changes on 7.x only.